kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/WatchConnectionManager.java

```diff
@@ -293,7 +293,7 @@ public void onPong(Buffer buffer) {
 
       @Override
       public void onClose(final int code, final String reason) {
-        logger.info("WebSocket close received. code: {}, reason: {}", code, reason);
```

Comment:
Thanks @ash211.
we want the same change in WatchHTTPManager.java as well I think.